src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -177,6 +177,21 @@ def register_options(cls, register):
                 "by the tests that were run."
             ),
         )
+        register(
+            "--fail-under",
+            type=float,
+            default=None,
+            help=(
+                "Fail if the total combined coverage percentage for all tests is less than this "
+                "number.\n\nUse this instead of setting fail_under in a coverage.py config file, "
```

Comment:
(No need for any change - this is solely an FYI for the future of something that I've found useful.) 

Recently I've been putting starting new paragraphs on new lines in the code. For example:

```python
"Fail if the total combined coverage percentage for all tests is less than this "
"number.\n\n"
"Use this instead"
```

I personally like it because it makes it easier to see the distinct paragraphs + reduces code diff when you want to change only one paragraph. Again, only an FYI.